src/cli/templates/init/botfile.js

```diff
@@ -1,13 +1,20 @@
 const isProd = process.env.NODE_ENV === 'production'
 const port = process.env.BOTPRESS_PORT || process.env.PORT || 3000
-const botUrl = isProd ? 'https://my-host.com' : 'http://localhost:' + port
+const botUrl = process.env.BOTPRESS_URL || `http://localhost:${port}`
```

Comment:
Ah, careful here. This is the issue I've fixed not long ago: https://github.com/botpress/botpress/commit/18388fed6482c421675a58bc551ed0be41d1667b (see the commit message)